src/Http/Response.php

```diff
@@ -35,7 +35,7 @@
 /**
  * Responses contain the response text, status and headers of a HTTP response.
  *
- * There are external packages such as `http-message-util` that provide HTTP
+ * There are external packages such as `fig/http-message-util` that provide HTTP
```

Comment:
Should be `psr` not `fig`.

Reply:
Is it?  I thought I pulled the official name.  I'll double-check.

Reply:
It is fig/http-message-util for the constants not the interface.  Am I missing something?

Reply:
My mistake, seems they use `psr` only for standards packages and `fig` for others.

Reply:
Quite the bad behavior and antipattern to not match the URL:
https://github.com/php-fig/http-message-util

The correct naming would IMO be `php-fig/http-message-util` - but meeeh